ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -152,6 +152,53 @@ function upgrade() {
     set +x
 }
 
+function upgrade_paused() {
+    set -x
+    unset TEST_SUITE
+    TARGET_RELEASES="${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}"
+    if [[ -f "${SHARED_DIR}/override-upgrade" ]]; then
+        TARGET_RELEASES="$(< "${SHARED_DIR}/override-upgrade")"
+        echo "Overriding upgrade target to ${TARGET_RELEASES}"
+    fi
+    # Split TARGET_RELEASES by commas, producing two releases
+    OPENSHIFT_UPGRADE0_RELEASE_IMAGE_OVERRIDE="$(echo $TARGET_RELEASES | cut -f1 -d,)"
+    OPENSHIFT_UPGRADE1_RELEASE_IMAGE_OVERRIDE="$(echo $TARGET_RELEASES | cut -f2 -d,)"
+
+    oc patch mcp/worker --type merge --patch '{"spec":{"paused":true}}'
+
+    echo "Starting control-plane upgrade to ${OPENSHIFT_UPGRADE0_RELEASE_IMAGE_OVERRIDE}"
+    openshift-tests run-upgrade "${TEST_UPGRADE_SUITE}" \
+        --to-image "${OPENSHIFT_UPGRADE0_RELEASE_IMAGE_OVERRIDE}" \
+        --options "${TEST_UPGRADE_OPTIONS-}" \
+        --provider "${TEST_PROVIDER}" \
+        -o "${ARTIFACT_DIR}/e2e.log" \
+        --junit-dir "${ARTIFACT_DIR}/junit" &
+    wait "$!"
+    echo "Upgraded control-plane to ${OPENSHIFT_UPGRADE0_RELEASE_IMAGE_OVERRIDE}"
+
+    echo "Starting control-plane upgrade to ${OPENSHIFT_UPGRADE1_RELEASE_IMAGE_OVERRIDE}"
+    openshift-tests run-upgrade "${TEST_UPGRADE_SUITE}" \
+        --to-image "${OPENSHIFT_UPGRADE1_RELEASE_IMAGE_OVERRIDE}" \
+        --options "${TEST_UPGRADE_OPTIONS-}" \
+        --provider "${TEST_PROVIDER}" \
+        -o "${ARTIFACT_DIR}/e2e.log" \
+        --junit-dir "${ARTIFACT_DIR}/junit" &
+    wait "$!"
+    echo "Upgraded control-plane to ${OPENSHIFT_UPGRADE1_RELEASE_IMAGE_OVERRIDE}"
+
+    echo "Starting worker upgrade to ${OPENSHIFT_UPGRADE1_RELEASE_IMAGE_OVERRIDE}"
+    oc patch mcp/worker --type merge --patch '{"spec":{"paused":false}}'
+    openshift-tests run-upgrade all \
+        --to-image "${OPENSHIFT_UPGRADE1_RELEASE_IMAGE_OVERRIDE}" \
```

Comment:
I don't understand this third `run-upgrade`, because we're already at `UPGRADE1`.  I would expect the third phase to be:

1. Unpause the `worker` pool (you do this above).
2. Wait for the `worker` pool to level.
3. (optional) Run e2e-parallel or some other non-update conformance suite.

Reply:
>Wait for the worker pool to level.
>(optional) Run e2e-parallel or some other non-update conformance suite.

This does exactly that - waits for worker pool update to complete and run disruption tests to ensure user workloads are not affected. Once we have a good passing rate on this I'll add minimal parallel conformance test